src/OrchardCore.Modules/OrchardCore.MiniProfiler/Manifest.cs

```diff
@@ -4,7 +4,20 @@
     Name = "Mini Profiler",
     Author = ManifestConstants.OrchardCoreTeam,
     Website = ManifestConstants.OrchardCoreWebsite,
-    Version = ManifestConstants.OrchardCoreVersion,
-    Description = "Adds the Mini Profiler widget to pages.",
-    Category = "Performance"
+    Version = ManifestConstants.OrchardCoreVersion
 )]
+
+[assembly: Feature(
+    Id = "OrchardCore.MiniProfiler",
+    Name = "Mini Profiler",
+    Category = "Performance",
+    Description = "Adds the Mini Profiler widget to pages."
+)]
+
+[assembly: Feature(
+    Id = "OrchardCore.MiniProfiler.Admin",
+    Name = "Mini Profiler Admin",
+    Category = "Performance",
+    Description = "Adds the Mini Profiler widget to both admin and front end pages.",
+    Dependencies = new[] { "OrchardCore.MiniProfiler" }
+)]
```

Comment:
Newline missing. I'm not too keen on it but as I was previously made aware it's necessary here :).

Reply:
It's an ancient tradition that we proudly honor and will pass down for future generations: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file